lib/Encoder.ts

```diff
@@ -13,6 +13,15 @@ export default class Encoder {
     return encodedContent;
   }
 
+  /**
+   * Decodes the given Base64URL string into a Buffer.
+   *
+   * Turns the encoded string to regular base 64 and decode as buffer
```

Comment:
Nit: avoid implementation detail in summary, consider remove or move it to above line 22.